### PR TITLE
Fix for JENKINS-52734 (JEP-200)

### PR DESCRIPTION
**Pull request by dglugla**
Update Maven's developers list

### CODE REVIEW

File: src/main/java/com/dynatrace/jenkins/dashboard/TABuildWrapper.java [hunk 1]
```diff
@@ -99,17 +99,13 @@ public TABuildWrapper(final String systemProfile, final String versionMajor, fin
...
 	@DataBoundConstructor
 	public TABuildWrapper(String systemProfile) {
-		this.systemProfile = systemProfile;
-		this.versionMajor = "";
-		this.versionMinor = "";
-		this.versionRevision = "";
-		this.versionMilestone = "";
-		this.marker = "";
-		this.recordSession = false;
+		this(systemProfile,"","","","","",false);
```
Comment (chojna) [inline src/main/java/com/dynatrace/jenkins/dashboard/TABuildWrapper.java]: spaces after commas :)

File: src/main/java/com/dynatrace/jenkins/dashboard/TABuildWrapper.java [hunk 2]
```diff
@@ -99,17 +99,13 @@ public TABuildWrapper(final String systemProfile, final String versionMajor, fin
 		this.versionMilestone = versionMilestone;
 		this.marker = marker;
 		this.recordSession = recordSession;
+
```
Comment (lugowskii) [inline src/main/java/com/dynatrace/jenkins/dashboard/TABuildWrapper.java]: new line not needed :)

Comment (dglugla) [reply src/main/java/com/dynatrace/jenkins/dashboard/TABuildWrapper.java]: not needed in fact, but nicely separates fields initialization from AppMon client creation

**Comment by oleg-nenashev** [comment]
The fix will be unlikely working well after the object is loaded from the disk. "sessions" will be null, and operations on it in the code will lead to NPEs

**Pull request merged**